internal/driver/interactive.go

```diff
@@ -267,6 +273,9 @@ func parseCommandLine(input []string) ([]string, variables, error) {
 		}
 	}
 	if c == nil {
+		if v := pprofVariables[name]; v != nil {
+			return nil, nil, fmt.Errorf("do you mean: \"%v=%v\"?", name, args[0])
```

Comment:
Do you really need %v here? Would %s be sufficient? Avoid %v unless required, it can leak implementation details in a funny way.

Reply:
Done.